### PR TITLE
luci-mod-admin-full: Add meshId support

### PR DESCRIPTION
**Pull request by kishangondaliya**
Add meshId support
[https://github.com/openwrt/luci/issues/1228](url)

Signed-off-by: Kishan Gondaliya <kishanpgondaliya@gmail.com>

### CODE REVIEW

**Comment by kishangondaliya** [comment]
@jow- @hnyman can you please look for this.

**Comment by kishangondaliya** [comment]
Does it require anything more? @jow- @hnyman

**Comment by SvenRoederer** [comment]
this references to #1228

**Comment by bobster-galore** [comment]
S.a. https://github.com/freifunk-berlin/firmware/commit/dd5f5aa690daba49da791b04cde982916b3f05e4#commitcomment-25259284 this patch makes 802.11s configurable in Luci and working.

**Comment by kishangondaliya** [comment]
@jow- @hnyman , follow-up on this PR.

**Comment by kishangondaliya** [comment]
@aparcar , @felixonmars can you please check this PR.

File: modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi.lua [hunk 1]
```diff
@@ -356,6 +356,20 @@ mode:value("ap", translate("Access Point"))
 mode:value("sta", translate("Client"))
 mode:value("adhoc", translate("Ad-Hoc"))
 
+ssid = s:taboption("general", Value, "ssid", translate("<abbr title=\"Extended Service Set Identifier\">ESSID</abbr>"))
```
Comment (jow-) [inline modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi.lua]: Why are you replicating the SSID option here? It is already declared a few lines up. Please add the extra dependencies to the existing `ssid` option object instead.

**Comment by kishangondaliya** [comment]
Hi @jow- ,
Have added requested changes. Kindly review it.

- Kishan Go.

File: modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi.lua (hunk 1 above)
Comment (kishangondaliya) [reply modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi.lua]: Hi @jow- ,
Have added requested changes. Kindly review it.

    Kishan Go.

**Comment by bobster-galore** [comment]
Would there be a review of the changed code this year perhaps?

**Comment by jow-** [comment]
I will not get around to this before January

**Comment by torte71** [comment]
I wonder about the order in your change: https://github.com/openwrt/luci/pull/1363/commits/b79806d600c9d5abd62767410bef5cc9e78c8c01

In (the new) lines 352..359 the "depends" for the "ssid" entry refer to the value of "mode".
But "mode" is set afterwards (line 361).
Shouldn't the "depends" hunk be applied after line 361 (i.e. after assigning "mode")?
Or do I just misunderstand the code?

**Comment by kishangondaliya** [comment]
I know the behavior and I would also like to change the sequence. But the thing is even after this much of time, this PR is still open, I really don't understand why??? If it got merged, I will add new PR for mesh params changes which I have already planned.

**Comment by jow-** [comment]
Merged via 267bf83db - thank you!

**Pull request closed without merging**